components/organisms/MarkdownRenderer.tsx

```
@@ -0,0 +1,68 @@
+        allowedAttributes: {
+            ...sanitizeHtml.defaults.allowedAttributes,
+            img: ['src', 'alt'],
+            span: ['class'],
+            code: ['class']
+        }
+    })
+
+    return (
+        <div
+            className="markdown-container prose mx-auto max-w-6xl px-6 dark:prose-invert lg:prose-xl"
+            dangerouslySetInnerHTML={{ __html: cleanHtml }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)